libraries/shared/storage/backfill/storage_value_loader_test.go

```diff
@@ -122,15 +138,33 @@ var _ = Describe("StorageValueLoader", func() {
 		Expect(bc.BatchGetStorageAtCalls).To(ConsistOf(
 			fakes.BatchGetStorageAtCall{BlockNumber: bigIntBlockOne, Account: addressOne, Keys: []common.Hash{keyOne}},
 			fakes.BatchGetStorageAtCall{BlockNumber: bigIntBlockOne, Account: addressTwo, Keys: []common.Hash{keyTwo}},
+			fakes.BatchGetStorageAtCall{BlockNumber: bigIntBlockOne, Account: addressThree, Keys: []common.Hash{keyThree}},
 		))
 	})
 
+	It("chunks requests to avoid 413 Request Entity Too Large reply from server", func() {
+		manyKeys := make([]common.Hash, 401)
+		for index, _ := range manyKeys {
+			manyKeys[index] = test_data.FakeHash()
+		}
+		keysLookupTwo.KeysToReturn = manyKeys
+
+		runnerErr := runner.Run()
+		Expect(runnerErr).NotTo(HaveOccurred())
+		Expect(keysLookupOne.GetKeysCalled).To(BeTrue())
+		Expect(keysLookupTwo.GetKeysCalled).To(BeTrue())
+
+		Expect(bc.BatchGetStorageAtCalls).To(ContainElement(fakes.BatchGetStorageAtCall{BlockNumber: bigIntBlockOne, Account: addressTwo, Keys: manyKeys[0:400]}))
+		Expect(bc.BatchGetStorageAtCalls).To(ContainElement(fakes.BatchGetStorageAtCall{BlockNumber: bigIntBlockOne, Account: addressTwo, Keys: manyKeys[400:]}))
```

Comment:
Would we also expect BatchGetStorageAtCalls to contain the call data for `keysLookupOne` as well?

Reply:
Yeah - everything should look basically the same as the test directly above this one except that the calls for `addressTwo` will be split into two chunks. Went back and forth about modifying the expectation for that test vs adding a new one, but figured adding a new one gave me a chance to document the specific HTTP error I saw in the docstring